views/js/controller/tests/editTest.js

```diff
@@ -1,12 +1,40 @@
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2015-2019 (original work) Open Assessment Technologies SA;
+ */
 
-define(['module', 'jquery','ui/lock'],
-	function(module, $, lock){
+/**
+ * Test edition controller
+ *
+ */
+define([
+    'jquery',
+    'ui/lock'
+],	function($, lock){
+    'use strict';
 
-        var editTestController = {
-            start : function(options){
-            	$('#lock-box').each(function() {lock($(this)).register()});
-            }
-        };
+    return {
 
-        return editTestController;
+        /**
+         * Controller's entrypoint
+         */
+        start(){
+            $('#lock-box').each(function() {
```

Comment:
Are there multiple `#lock-box`?

Reply:
it shouldn't but I won't change the behavior in a pr which is a syntax update